plugins/ua_config_standard.c

```diff
@@ -9,6 +9,7 @@
 #include "ua_securitypolicy_none.h"
 #include "ua_types.h"
 #include "ua_types_generated_handling.h"
+#include "ua_client_highlevel.h"
```

Comment:
Though it might be worth a thought to split the default client and default server config into two separate files.